src/popup/router/routes/index.js

```diff
@@ -326,10 +327,27 @@ export default [
     },
   },
   {
-    path: '/tokens',
-    component: Tokens,
+    name: 'tokens-preview',
+    path: '/tokens-preview',
+    component: TokensPreview,
     meta: {
-      title: 'fungible-tokens',
+      title: 'tokens-preview',
+    },
+  },
+  {
+    name: 'token-details',
+    path: '/tokens-preview/:contract',
+    component: TokenDetails,
+    meta: {
+      title: 'token-details',
+      notPersist: true,
+    },
+    beforeEnter: (to, from, next) => {
+      if (to.name === 'token-details' && from.name !== 'tokens-preview') {
```

Comment:
it should be possible to open token details not only from the token preview page